### PR TITLE
Image pruner

### PR DESCRIPTION
**Pull request by Talits**
add image pruner

### CODE REVIEW

File: ansible/playbooks/getup-monitoring/config.yml [hunk 1]
```diff
@@ -19,7 +19,9 @@
         name: getup-prometheus
     - include_role:
         name: getup-grafana
-
+    - include_role:
```
Comment (caruccio) [inline ansible/playbooks/getup-monitoring/config.yml]: O imagepruner nao tem relação com o prometheus. O ideal seria ter uma task para ele.

Comment (Talits) [reply ansible/playbooks/getup-monitoring/config.yml]: Done ! I separated the playbook !

File: ansible/roles/getup-imagepruner/templates/imagepruner.j2 [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+            name: image-pruner
+            env:
+              - name: KEEP_COMPLETE
+                value: "10"
+              - name: KEEP_FAILED
+                value: "10"
+              - name: KEEP_TAGS
+                value: "10"
+              - name: KEEP_YOUNGER
+                value: "720h"
```
[the remark this answers is not in the document]
Comment (Talits) [reply ansible/roles/getup-imagepruner/templates/imagepruner.j2]: ok

File: ansible/roles/getup-prometheus/templates/prometheus.getup.rules.j2 [hunk 3]
```diff
@@ -357,6 +357,7 @@ groups:
     annotations:
```
Comment (caruccio) [inline ansible/roles/getup-prometheus/templates/prometheus.getup.rules.j2]: This file has nothing to do with imagepruner. Should be removed from PR.
Tip: create a separated branch in your repo to hold related commits only.

**Pull request closed without merging**